runtime/tr.source/trj9/z/codegen/J9S390CHelperLinkage.cpp

```diff
@@ -135,7 +138,14 @@ TR::S390CHelperLinkage::S390CHelperLinkage(TR::CodeGenerator * codeGen,TR_S390Li
       {
       // 31-Bit zOS will need GPR12 for CAA register so it won't be preserved. For all other variant it is preserved
       if (TR::Compiler->target.is64Bit())
-         setRegisterFlag(TR::RealRegister::GPR12, Preserved); 
+         {
```

Comment:
GPR14 is a preserved register according to XPLINK register linkage.  Should that be included here?

Reply:
@joransiu Good catch. Just checked linkage, We do need to preserve it on zOS. Added it. Let me do basic test with it.

Reply:
Finished local unit tests. Should be good to go.